lib/bitcoin.py

```diff
@@ -337,7 +337,10 @@ def script_to_p2wsh(script):
 def address_to_script(addr):
     if is_segwit_address(addr):
         witver, witprog = segwit_addr.decode(SEGWIT_HRP, addr)
-        script = bytes([witver]).hex() + push_script(bytes(witprog).hex())
+        assert (0 <= witver <= 16)
+        OP_n = (witver+0x50) * (witver > 0)
```

Comment:
This gets the job done but is hard to read if you don't understand the way a boolean resolved in a mathmatic operation.

I suggest:

```python
OP_n = witver + 0x50 if witver > 0 else 0
```

Reply:
I didn't know python had a ternary or I would have used it!